src/main/java/uk/gov/hmcts/divorce/caseworker/service/IssueApplicationService.java

```diff
@@ -57,11 +59,17 @@
 
     public CaseDetails<CaseData, State> issueApplication(final CaseDetails<CaseData, State> caseDetails) {
         return caseTasks(
-            setPostIssueState,
             details -> {
-                details.getData().getApplication().setIssueDate(LocalDate.now(clock));
+                final CaseData caseData = details.getData();
+                final Applicant applicant1 = caseData.getApplicant1();
+                final Applicant applicant2 = caseData.getApplicant2();
+                if (!applicant1.isRepresented() && !applicant2.isRepresented() && applicant2.isBasedOverseas()) {
+                    caseData.getApplication().setSolServiceMethod(PERSONAL_SERVICE);
+                }
+                caseData.getApplication().setIssueDate(LocalDate.now(clock));
```

Comment:
Move this to two tasks SetServiceType, SetIssueDate

Reply:
Complete
https://github.com/hmcts/nfdiv-case-api/pull/1671/commits/e40b081b11eabec02324d47ba9208bd6282ee016